File: src/main/java/pe/nanamochi/banchus/packets/client/MatchCompletePacket.java
package pe.nanamochi.banchus.packets.client;

import lombok.Data;
import lombok.NoArgsConstructor;
import pe.nanamochi.banchus.packets.Packet;
import pe.nanamochi.banchus.packets.Packets;

@Data
@NoArgsConstructor
public class MatchCompletePacket implements Packet {
  @Override
  public Packets getPacketType() {
    return Packets.OSU_MATCH_COMPLETE;
  }
}

File: banchus-packet/src/main/kotlin/pe/nanamochi/banchus/packets/client/MatchCompletePacket.kt
package pe.nanamochi.banchus.packets.client

import java.io.InputStream
import pe.nanamochi.banchus.core.ClientPacket
import pe.nanamochi.banchus.core.PacketType
import pe.nanamochi.banchus.io.DataReader

class MatchCompletePacket : ClientPacket() {
    override val type = PacketType.OSU_MATCH_COMPLETE

    override fun read(reader: DataReader, ins: InputStream) {}
}
